common/config/babel/features.js

```diff
@@ -1,9 +1,38 @@
+/// A list of features recognized by the conditional compilation preprocessing plugin.
+///
+/// alpha - Features that are in development and not yet ready for general use.
+///         These features conditional-compiled out of beta and stable npm packages.
+/// beta - Features that are in development and are ready for public preview.
+///        These features are conditional-compiled out of stable npm packages.
+/// stable - Features that are ready for general use. These features are included in all releases.
+///
 module.exports = {
-  // A list of features recognized by the conditional compilation preprocessing plugin.
-  // "demo" is a special feature, used for demo purposes. For this feature,
-  // The plugin removes any AST node that is preceded by a comment that contains the tag:
-  // @conditional-compile-remove(demo)
-  "features": [
+  alpha: [
```

Comment:
Will the code in  
```
function createFeatureSet(features, inProgressFeatures) {
  const featureSet = {}
  if(inProgressFeatures !== undefined) {
    features.forEach(f => featureSet[`@conditional-compile-remove(${f})`] = inProgressFeatures.includes(f));
  } else {
    features.forEach(f => featureSet[`@conditional-compile-remove(${f})`] = true);
  }
  
  return featureSet;
}
```
still work with this change?

Reply:
Thanks for checking this! Yep in this case both features and inProgressFeatures are named passed in args